src/models/db.js

```diff
@@ -33,6 +51,45 @@ module.exports = function() {
     },
     phone: {
       type: Sequelize.STRING
+    },
+
+    //added ability for user to set gender in DB on registration to generate recommendations based on gender
```

Comment:
The comments on this file, while robust, do not need to explain the context in which the code was added. Comments need only be applied for two major reasons. 

1. If the code is sufficiently complex and needs to be explained
1. If the documentation engine (jsdocs) requires it